documentation/architecture/README.md

```diff
@@ -11,234 +11,23 @@ This is a general architecture diagram of the simpler.grants.gov system.
-    end
-
-    subgraph AZ1 [Availability zone 1]
-        direction TB
-        private-subnet1:::subnet
-        subgraph private-subnet1 [Private Subnet]
-            ECS1:::ecs
-            ECS2:::ecs
-        end
-
-    end
-    subgraph AZ2 [Availability zone 2]
-        direction TB
-        private-subnet2:::subnet
-        subgraph private-subnet2 [Private Subnet]
-            ECS3:::ecs
-            ECS4:::ecs
-        end
-
-    end
-    subgraph ECS1 ["ECS Front-End Task (Next.js)"]
-        c1[" Docker
-        Container 1"]
-        c2[" Docker
-        Container n"]
-    end
-    subgraph ECS2 ["ECS Back-End Task (Flask)"]
-        c3[" Docker
-        Container 1"]
-        c4[" Docker
-        Container n"]
-    end
-    subgraph ECS3["ECS Front-End Task (Next.js)"]
-        c5[" Docker
-        Container 1"]
-        c6[" Docker
-        Container n"]
-    end
-    subgraph ECS4 ["ECS Back-End Task (Flask)"]
-        c7[" Docker
-        Container 1"]
-        c8[" Docker
-        Container n"]
-    end
-
-    subgraph RDS
-        subgraph private-subnet3 [Private Subnet]
-            DB[("Multi-AZ Grants.gov
-            Replica Database")]:::db
-            PDB[("Multi-AZ Postgres DB
-            for Back End")]:::db
-        end
-    end
-    style RDS stroke:blue,color:blue
-    private-subnet3:::subnet
-
-    ecr --> ECSC
-    ECSS --> AZ1 & AZ2
-    public[Public Internet Users] --> ALB
-
     %% CI/CD Pipeline
     eng["Developers fas:fa-laptop-code"] --"Push to main branch fas:fa-code-branch"--> GH
     subgraph GH ["Github fab:fa-github"]
         repo[Simpler Grants Repo]
         click repo href "https://github.com/HHS/simpler-grants-gov" _blank
     end
     GH --Build and Deploys Image--> iam --> ecr
-    GH --Restarts task with new Image--> iam --> ECSS
-
-
-    %% Styles
-    classDef ecs fill:#FF9900,color:black
-    classDef db fill:blue,color:white
-    classDef az color:green,stroke:green,stroke-dasharray: 5 5
-    classDef subnet color:darkblue, stroke:darkblue, stroke-dasharray: 5 5
-    classDef lb fill:purple, color:white
-    classDef sec fill:red,color:white
-
+    GH --Restarts task with new Image--> iam --> ECS
 ```
 
 ## AWS Hosted Infrastructure
 
 This is an architecture diagram focusing on the AWS shared infrastructure managed by simpler.grants.gov
 
-```mermaid
-%%{init: {'theme': 'neutral' } }%%
-flowchart TD
-        %% AWS Tenant
-    subgraph AWS [HHS AWS Tenant]
-        VPC2:::az
-    end
-
-    %% AWS Services Within VPC
-    subgraph VPC2 ["AWS Virtual Private Cloud (VPC)"]
-        direction LR
-        AZ1:::az
-        AZ2:::az
-        public-subnet1:::subnet
-        RDS
-        AZ1 & AZ2 --> RDS
-    end
-
-    subgraph public-subnet1 [Public Subnet]
-        ALB["Application Load Balancer (ALB)"]:::lb --> AZ1 & AZ2
-    end
+![Grants gov system architecture - v1 0 0 (current) (4)](https://github.com/HHS/simpler-grants-gov/assets/5768468/a7e8adb5-5e23-4623-9bda-66e34f437079)
```

Comment:
(question): Is Route 53 involved in the DNS resolution if we aren't using it as a service?  
![image](https://github.com/HHS/simpler-grants-gov/assets/512243/3659eb70-a45b-42ba-9e08-29b42c52c3ed)

Reply:
Oh, good catch!

It might be used "under the hood" by the load balancers, but we aren't using it directly. Will remove 👍

Reply:
@coilysiren (nit) we aren't using New Relic yet, so should note that in the diagram or description.

Reply:
Just for correctness, I'll add a version of the diagram to this readme that doesn't have new relic.